share/transformers/com_mendeley_data.py

```diff
@@ -0,0 +1,435 @@
+from share.transform.chain import *  # noqa
+from share.transform.chain.utils import format_address
+
+
+def format_mendeley_address(ctx):
+    return format_address(
+        address1=ctx['name'],
+        city=ctx['city'],
+        state_or_province=ctx['state'],
+        country=ctx['country']
+    )
+
+RELATION_MAP = {
+    'related_to': '',
+    'derived_from': 'IsDerivedFrom',
+    'source_of': 'IsDerivedFrom',
+    'compiles': 'Compiles',
+    'compiled_by': 'Compiles',
+    'cites': 'Cites',
+    'cited_by': 'Cites',
+}
+
+INVERSE_RELATIONS = (
+    'cited_by',
+    'compiled_by',
+    'derived_from',
+)
+
+RELATIONS = (
+    'cites',
+    'compiles',
+    'source_of',
+    'related_to',
+)
+
+
+def get_related_works(options, inverse):
+    results = []
+    for option in options:
+        relation = option['rel']
+        if inverse and relation in INVERSE_RELATIONS:
+            results.append(option)
+        elif not inverse and relation in RELATIONS:
+            results.append(option)
+    return results
+
+
+def get_relation_type(relation_type):
+    normalized_relation = RELATION_MAP[relation_type]
+    return normalized_relation or 'WorkRelation'
+
+
+class WorkIdentifier(Parser):
+    uri = ctx
```

Comment:
It is. I put the IRI parser in the [creativework](https://github.com/CenterForOpenScience/SHARE/pull/655/files#diff-fbb6801351f10ca0870441afa2554c4aR388) parser because if it gets to the the workidentifier and then fails, the whole record fails validation.